css/pages/home.css

```diff
@@ -10,4 +11,14 @@
 @import '../components/button.css';
 
 /* animations */
-@import '../base/anim/keyframes.scss';
\ No newline at end of file
+@import '../base/anim/keyframes.scss';
+
+/* Header */
```

Comment:
Ar nereikia po kiekvienos sekcijos palikti tarpo, kad nebūtų konfliktų? 
Nes pagal logiką pvz.: Hero sekcijos kodas bus rašomas ant 18 eilutės kurioje jau yra kažkas. 
Jeigu klystu, pakomentuokit.

Reply:
Manau tarpai konfliktų išvengimui nepadėtų, nes tarpas nėra stabili eilutė. Stabili, nekintanti eilutė yra komentarai. Tai jie kaip ir sudėti, tik manau būtų geriau jei komentarais pažymėtume sekcijos pradžią ir pabaigą. Tuomet turėtume nekintamą sekciją, kodo blokui parašyti. 
Jei klystu, taip pat pataisykite :)

Reply:
Taip, Simona teisi: reikia pažymėti komentaru pradžią ir pabaigą. Pataisiau. Bet taip pat pastebėjau, kad index.html komentarai sekcijoms taip pat be pabaigos komentaro. Galbūt pataisys, kas kaip tik dirba su tuo failu?